common/src/main/java/com/tc/lang/TCThreadGroup.java

```diff
@@ -105,6 +106,24 @@ public boolean retire(long timeout, Consumer<InterruptedException> interruptHand
     return complete;
   }
 
+  private boolean lookForThreadExit(Thread t, Consumer<InterruptedException> interruptHandler) {
+    if (ignorePoolThreads && (t.getName().startsWith("pool-") || t.getName().startsWith("ForkJoinPool.commonPool-worker-"))) {
```

Comment:
`t instanceof ForkJoinWorkerThread` would at least make this slightly less ugly.

Reply:
that would mean any fork join worker thread.  not just the common pool.

Reply:
Sorry...:
```
(t instanceof ForkJoinWorkerThread) && ((ForkJoinWorkerThread) t).getPool() == ForkJoinPool.commonPool()
```